src/views/editor/gutter.rs

```diff
@@ -152,14 +184,44 @@ impl Widget for EditorGutterView {
                 let mut text_layout = TextLayout::new();
                 if line == current_line {
                     text_layout.set_text(&text, current_line_attrs_list.clone());
+                    if let Some(current_line_color) = self.gutter_style.current_line_color() {
```

Comment:
Added the ability to have the gutter show the highlighted line. This appears continuous with the editor highlighted line.